Shared with others list

LukasHirt opened the pull request:
## Description
<!--- Describe your changes in detail -->
Created `SharedFilesList` component. Moved most of file list related functionality into mixins so we can reuse it in different lists. At the moment this list doesn't contain any actions thus checkboxes and star were removed as well and file name got class `uk-disabled` to prevent any hover/focus on it and to allow selecting of the row by clicking on it.

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->
- Refs #1186 

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
- test environment: Manually
1. Share file with other user
2. Share file that has been shared with me with can share permission with some other user

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/25989331/62855889-ae5e4800-bcf3-11e9-880d-e61ef1f3ac54.png)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
<!-- Tick the checkboxes when done. -->
<!-- Raise documentation ticket in https://github.com/owncloud/documentation -->
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

DeepDiver1975 commented on apps/files/src/default.js:
```diff
@@ -56,6 +56,14 @@ const navItems = [
     }
   },
+  {
+    name: 'Shared with others',
```
we need to find a way to get this translated ....

LukasHirt replied:
I guess we could translate it inside the menu component in computed properties but doesn't really seem like a nice solution.

DeepDiver1975 commented on apps/files/src/store/actions.js:
```diff
@@ -248,6 +278,27 @@ export default {
+  loadFolderSharedFromMe (context, { client, $gettext }) {
+    context.commit('UPDATE_FOLDER_LOADING', true)
+
+    client.requests.ocs({
```
are we missing an api method in the sdk here?

LukasHirt replied:
I found mentions of shared_with_me in the shares there but no way how to load shared with others. (tried to use the getShares function, it's possible I overlooked something or did something wrong though)

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4572/10

<p>


![20190806-154339-498.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154339-498.png)
![20190806-154339-914.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154339-914.png)
![20190806-154340-416.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154340-416.png)
![20190806-154418-042.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154418-042.png)
![20190806-154418-379.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154418-379.png)
![20190806-154456-601.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154456-601.png)
![20190806-154456-881.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154456-881.png)
![20190806-154532-089.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154532-089.png)
![20190806-154532-451.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154532-451.png)
![20190806-154621-893.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154621-893.png)
![20190806-154701-613.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154701-613.png)
![20190806-154759-516.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-154759-516.png)
![20190806-155011-644.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155011-644.png)
![20190806-155132-485.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155132-485.png)
![20190806-155209-810.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155209-810.png)
![20190806-155256-675.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155256-675.png)
![20190806-155344-821.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155344-821.png)
![20190806-155432-268.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155432-268.png)
![20190806-155525-516.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155525-516.png)
![20190806-155618-380.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155618-380.png)
![20190806-155800-670.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155800-670.png)
![20190806-155844-740.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155844-740.png)
![20190806-155928-701.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-155928-701.png)
![20190806-160012-243.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160012-243.png)
![20190806-160057-998.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160057-998.png)
![20190806-160149-882.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160149-882.png)
![20190806-160150-177.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160150-177.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4572/13

<p>


![20190806-160108-051.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160108-051.png)
![20190806-160108-226.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160108-226.png)
![20190806-160108-499.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160108-499.png)
![20190806-160156-464.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160156-464.png)
![20190806-160156-648.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160156-648.png)
![20190806-160156-775.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160156-775.png)
![20190806-160341-167.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160341-167.png)
![20190806-160537-220.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160537-220.png)
![20190806-160644-309.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160644-309.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4572/12

<p>


![20190806-160107-712.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160107-712.png)
![20190806-160107-987.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160107-987.png)
![20190806-160108-320.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160108-320.png)
![20190806-160207-923.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160207-923.png)
![20190806-160208-130.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160208-130.png)
![20190806-160208-312.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160208-312.png)
![20190806-160351-831.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160351-831.png)
![20190806-160443-789.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160443-789.png)
![20190806-160533-217.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160533-217.png)
![20190806-160619-451.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-160619-451.png)
![20190806-161052-920.png](https://minio.owncloud.com/phoenix/screenshots/4572/20190806-161052-920.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4648/

<p>


![20190812-102048-734.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102048-734.png)
![20190812-102049-333.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102049-333.png)
![20190812-102129-553.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102129-553.png)
![20190812-102129-950.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102129-950.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4648/

<p>


![20190812-102159-294.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102159-294.png)
![20190812-102159-610.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102159-610.png)
![20190812-102159-874.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102159-874.png)
![20190812-102240-615.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102240-615.png)
![20190812-102241-226.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102241-226.png)
![20190812-102241-682.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102241-682.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4648/

<p>


![20190812-102314-904.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102314-904.png)
![20190812-102315-405.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102315-405.png)
![20190812-102405-136.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102405-136.png)
![20190812-102405-416.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102405-416.png)
![20190812-102444-983.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102444-983.png)
![20190812-102445-253.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102445-253.png)
![20190812-102541-321.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102541-321.png)
![20190812-102541-690.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102541-690.png)
![20190812-102624-338.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102624-338.png)
![20190812-102624-894.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102624-894.png)
![20190812-102706-476.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102706-476.png)
![20190812-102706-731.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102706-731.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4648/

<p>


![20190812-102613-289.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102613-289.png)
![20190812-102613-513.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102613-513.png)
![20190812-102651-086.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102651-086.png)
![20190812-102651-410.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102651-410.png)
![20190812-102728-626.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102728-626.png)
![20190812-102728-752.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102728-752.png)
![20190812-102800-031.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102800-031.png)
![20190812-102800-402.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102800-402.png)
![20190812-102832-551.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102832-551.png)
![20190812-102832-816.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102832-816.png)
![20190812-102903-196.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102903-196.png)
![20190812-102903-414.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102903-414.png)
![20190812-102938-446.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102938-446.png)
![20190812-102938-570.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102938-570.png)
![20190812-103013-939.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103013-939.png)
![20190812-103014-173.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103014-173.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4648/

<p>


![20190812-102736-176.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102736-176.png)
![20190812-102736-274.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102736-274.png)
![20190812-102819-147.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102819-147.png)
![20190812-102819-217.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102819-217.png)
![20190812-102848-464.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102848-464.png)
![20190812-102848-533.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102848-533.png)
![20190812-102925-012.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102925-012.png)
![20190812-102925-089.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102925-089.png)
![20190812-102951-825.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102951-825.png)
![20190812-102951-893.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-102951-893.png)
![20190812-103024-377.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103024-377.png)
![20190812-103024-536.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103024-536.png)
![20190812-103103-563.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103103-563.png)
![20190812-103103-644.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103103-644.png)
![20190812-103139-160.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103139-160.png)
![20190812-103139-255.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103139-255.png)
![20190812-103225-562.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103225-562.png)
![20190812-103225-712.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103225-712.png)
![20190812-103225-862.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103225-862.png)
![20190812-103301-668.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103301-668.png)
![20190812-103301-757.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103301-757.png)
![20190812-103301-932.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103301-932.png)
![20190812-103416-509.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103416-509.png)
![20190812-103454-901.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103454-901.png)
![20190812-103545-527.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103545-527.png)
![20190812-103628-783.png](https://minio.owncloud.com/phoenix/screenshots/4648/20190812-103628-783.png)

</p></details>

DeepDiver1975 commented:
Is it possible to add a column with the recipients?

e.g. for user shares you have this information
![Screenshot from 2019-08-12 14-50-32](https://user-images.githubusercontent.com/1005065/62866242-96e18800-bd10-11e9-975a-fbb80de9cba1.png)

for link shares at least a chain icon can be displayed ...

LukasHirt commented:
> Is it possible to add a column with the recipients?

With the current implementation we could display only one recipient per item... I'm filtering the response to remove all duplicate files to save a little bit of the performance and not build the same file e. g. 5 times.

DeepDiver1975 commented:
> With the current implementation we could display only one recipient per item... I'm filtering the response to remove all duplicate files to save a little bit of the performance and not build the same file e. g. 5 times.

Understood - oc10 is doing the same afaik
![Screenshot from 2019-08-12 15-28-23](https://user-images.githubusercontent.com/1005065/62868417-e8d8dc80-bd15-11e9-9cc8-a94b8fcd9fad.png)

DeepDiver1975 commented:
and there is a conflict - please resolve - THX

LukasHirt commented:
> Understood - oc10 is doing the same afaik

I added at least that one user. We can try to figure this out later IMHO 🤷‍♂

> and there is a conflict - please resolve - THX

Rebased :wink:

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4660/

<p>


![20190812-134958-363.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134958-363.png)
![20190812-134958-931.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134958-931.png)
![20190812-135042-820.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135042-820.png)
![20190812-135043-188.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135043-188.png)
![20190812-135129-176.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135129-176.png)
![20190812-135129-474.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135129-474.png)
![20190812-135214-349.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135214-349.png)
![20190812-135214-734.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135214-734.png)
![20190812-135248-042.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135248-042.png)
![20190812-135248-410.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135248-410.png)
![20190812-135321-800.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135321-800.png)
![20190812-135322-081.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135322-081.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4660/

<p>


![20190812-134803-307.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134803-307.png)
![20190812-134803-530.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134803-530.png)
![20190812-134850-531.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134850-531.png)
![20190812-134851-116.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134851-116.png)
![20190812-134943-091.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134943-091.png)
![20190812-134943-577.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-134943-577.png)
![20190812-135029-992.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135029-992.png)
![20190812-135030-546.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135030-546.png)
![20190812-135122-319.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135122-319.png)
![20190812-135122-882.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135122-882.png)
![20190812-135212-124.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135212-124.png)
![20190812-135212-629.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135212-629.png)
![20190812-135245-075.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135245-075.png)
![20190812-135245-291.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135245-291.png)
![20190812-135328-395.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135328-395.png)
![20190812-135328-906.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135328-906.png)
![20190812-135408-417.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135408-417.png)
![20190812-135408-732.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135408-732.png)
![20190812-135449-479.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135449-479.png)
![20190812-135449-811.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135449-811.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4660/

<p>


![20190812-135333-481.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135333-481.png)
![20190812-135333-573.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135333-573.png)
![20190812-135422-443.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135422-443.png)
![20190812-135422-642.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135422-642.png)
![20190812-135529-138.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135529-138.png)
![20190812-135529-275.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135529-275.png)
![20190812-135609-619.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135609-619.png)
![20190812-135609-720.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135609-720.png)
![20190812-135656-199.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135656-199.png)
![20190812-135656-322.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135656-322.png)
![20190812-135741-107.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135741-107.png)
![20190812-135741-285.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135741-285.png)
![20190812-135819-664.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135819-664.png)
![20190812-135819-817.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135819-817.png)
![20190812-135902-999.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135902-999.png)
![20190812-135903-136.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135903-136.png)
![20190812-135955-844.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135955-844.png)
![20190812-135955-931.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135955-931.png)
![20190812-135956-080.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135956-080.png)
![20190812-140029-220.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140029-220.png)
![20190812-140029-352.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140029-352.png)
![20190812-140205-193.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140205-193.png)
![20190812-140345-471.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140345-471.png)
![20190812-140432-679.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140432-679.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4660/

<p>


![20190812-135546-771.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135546-771.png)
![20190812-135547-028.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135547-028.png)
![20190812-135628-918.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135628-918.png)
![20190812-135629-191.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135629-191.png)
![20190812-135704-312.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135704-312.png)
![20190812-135704-636.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135704-636.png)
![20190812-135751-176.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135751-176.png)
![20190812-135751-310.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135751-310.png)
![20190812-135837-886.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135837-886.png)
![20190812-135838-195.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135838-195.png)
![20190812-135915-188.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135915-188.png)
![20190812-135915-409.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135915-409.png)
![20190812-135954-896.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135954-896.png)
![20190812-135955-195.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135955-195.png)
![20190812-140034-812.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140034-812.png)
![20190812-140034-991.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140034-991.png)
![20190812-140116-043.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140116-043.png)
![20190812-140116-203.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140116-203.png)
![20190812-140116-372.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140116-372.png)
![20190812-140156-678.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140156-678.png)
![20190812-140156-847.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140156-847.png)
![20190812-140157-118.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140157-118.png)
![20190812-140248-049.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140248-049.png)
![20190812-140422-335.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140422-335.png)
![20190812-140516-733.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140516-733.png)
![20190812-140602-625.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140602-625.png)
![20190812-140644-681.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140644-681.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4660/

<p>


![20190812-135354-474.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135354-474.png)
![20190812-135354-799.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135354-799.png)
![20190812-135355-292.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135355-292.png)
![20190812-135438-426.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135438-426.png)
![20190812-135438-719.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135438-719.png)
![20190812-135438-988.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135438-988.png)
![20190812-135525-075.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135525-075.png)
![20190812-135525-445.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135525-445.png)
![20190812-135525-763.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135525-763.png)
![20190812-135601-848.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135601-848.png)
![20190812-135602-485.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135602-485.png)
![20190812-135641-503.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135641-503.png)
![20190812-135719-914.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135719-914.png)
![20190812-135935-497.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-135935-497.png)
![20190812-140039-367.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140039-367.png)
![20190812-140126-486.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140126-486.png)
![20190812-140231-026.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140231-026.png)
![20190812-140324-087.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140324-087.png)
![20190812-140431-121.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140431-121.png)
![20190812-140606-642.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140606-642.png)
![20190812-140649-382.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140649-382.png)
![20190812-140829-706.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140829-706.png)
![20190812-140915-322.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140915-322.png)
![20190812-140958-550.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-140958-550.png)
![20190812-141041-298.png](https://minio.owncloud.com/phoenix/screenshots/4660/20190812-141041-298.png)

</p></details>

DeepDiver1975 commented:
@LukasHirt whats wrong with the tests? mind having a look? THX

LukasHirt commented:
Should be green now. Let's see

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4662/

<p>


![20190812-152137-989.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152137-989.png)
![20190812-152138-148.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152138-148.png)
![20190812-152138-300.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152138-300.png)
![20190812-152213-613.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152213-613.png)
![20190812-152213-744.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152213-744.png)
![20190812-152213-951.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152213-951.png)
![20190812-152324-815.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152324-815.png)
![20190812-152506-745.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152506-745.png)
![20190812-152544-044.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152544-044.png)

</p></details>

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4662/

<p>


![20190812-152338-346.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152338-346.png)
![20190812-152338-453.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152338-453.png)
![20190812-152338-630.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152338-630.png)
![20190812-152403-352.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152403-352.png)
![20190812-152403-459.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152403-459.png)
![20190812-152514-501.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152514-501.png)
![20190812-152559-482.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152559-482.png)
![20190812-152654-628.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152654-628.png)
![20190812-152731-981.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152731-981.png)

</p></details>

LukasHirt commented:
Damn. Still not 🤬

ownclouders commented:
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4662/

<p>


![20190812-151529-771.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151529-771.png)
![20190812-151530-275.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151530-275.png)
![20190812-151530-649.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151530-649.png)
![20190812-151613-613.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151613-613.png)
![20190812-151613-904.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151613-904.png)
![20190812-151614-264.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151614-264.png)
![20190812-151706-087.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151706-087.png)
![20190812-151706-520.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151706-520.png)
![20190812-151706-809.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151706-809.png)
![20190812-151736-430.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151736-430.png)
![20190812-151736-921.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151736-921.png)
![20190812-151818-133.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151818-133.png)
![20190812-151900-241.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-151900-241.png)
![20190812-152008-395.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152008-395.png)
![20190812-152105-543.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152105-543.png)
![20190812-152244-103.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152244-103.png)
![20190812-152423-884.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152423-884.png)
![20190812-152511-041.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152511-041.png)
![20190812-152612-049.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152612-049.png)
![20190812-152714-646.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152714-646.png)
![20190812-152808-586.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152808-586.png)
![20190812-152853-579.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-152853-579.png)
![20190812-153034-102.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-153034-102.png)
![20190812-153119-545.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-153119-545.png)
![20190812-153208-366.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-153208-366.png)
![20190812-153252-978.png](https://minio.owncloud.com/phoenix/screenshots/4662/20190812-153252-978.png)

</p></details>

LukasHirt commented:
@DeepDiver1975 Green 🙌
